fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +222,43 @@ then
 	log_Err_Abort "the --usejacobian option must be 'true' or 'false'"
 fi
 
+
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+BOLDMask=`opts_GetOpt1 "--boldmask" $@`                                   # Specifies what mask to use for the final bold:
+                                                                          #   T1_fMRI_FOV: combined T1w brain mask and FOV masks (the default), 
```

Comment:
Quibble, but can we say "fMRI FOV" in each case, for consistency.
e.g.,
`# #   T1_fMRI_FOV: combined T1w brain mask and fMRI FOV (the default),`

Reply:
Sure. I like your quibbles.